plugins/file-open-save-plugin/impl/core/src/main/javascript/app/app.component.js

```diff
@@ -108,13 +109,13 @@ define([
     function setState(state) {
       if (state === "open") {
         vm.wrapperClass = "open";
```

Comment:
should be 
vm.headerTitle = i18n.get("file-open-save-plugin.app.header.open.title");
vm.confirmButton = i18n.get("file-open-save-plugin.app.open.button");